gcivil-nyu-org/team-4-inperson
# Logging

Created logging functionality. Also changed settings files to enable the same in dev and production environment.

Definition of done.
You pull the latest code, create the necessary file. 
And when you search something from index page. You can see the custom logs that are added in views.py of search.

How to test?
Refer the help doc created in google. 
https://docs.google.com/document/d/1G1Qsu4p0wC1RXvnHmYaMFUvujdaQcMvFwJLvzl2J3Is/edit?usp=sharing

I am not sure if logging will work in dev/prod. I have to check this when it is merged with develop.
